conda/cli/conda_argparse.py

```diff
@@ -1037,16 +1037,13 @@ def configure_parser_remove(sub_parsers, name='remove'):
 
 
 def configure_parser_run(sub_parsers):
-    help = "Run an executable in a conda environment. [Experimental]"
+    help = "Run an executable in a conda environment."
     descr = help + dedent("""
 
-    Use '--' (double dash) to separate CLI flags for 'conda run' from CLI flags sent to
-    the process being launched.
+    Example usage::
```

Comment:
I don't think double colon is needed here?

Reply:
This will make the code example render as a code block on https://continuumio-conda-docs--11302.com.readthedocs.build/projects/conda/en/11302/commands/run.html

Reply:
Ah mixing RST and help blurbs